.dvc/config

```diff
@@ -0,0 +1,4 @@
+['remote "metakb"']
+url = s3://metakb-dvc
```

Comment:
Are there any size restrictions here? IIRC, last time we were butting up against free-tier availability. As we expand the captured data, will that be a problem for this bucket? If we go this route, should we instead use the WUSTL-hosted S3 account for the VICC project?

Reply:
Yes, for sure we want to use an official WUSTL-hosted S3 account.